content/docs/use-cases/kubeflow-on-multinode-cluster.md

```diff
@@ -0,0 +1,251 @@
+
+In fact, in KubeFlow there are three pods that have 10Gi PVCs that will get stuck in pending state 
+if they are unable to bind their PVC. The three pods are minio, mysql, and vizier-db. 
+You can check the status of the PVC requests:
+
+```
+kubectl -n ${NAMESPACE} get pvc
+```
+
+  * Look for the status of "Bound"
+  * PVC requests in "Pending" state indicate that the scheduler was unable to bind the required PVC.
+
+In order to change status of PersistentVolumeClaims in "Bound" three Persistent Volumes (PVs) must be provisioned.
+
+But wait! 
+
+There are a couple of problems to solve before: you are in a multi-node cluster, you cannot simply create an *hostPath* PersistentVolume (the volume that exposes a filesystem directory to a Pod) like in a single node scenario.
+
+## A step back: the storage on Kubernetes
+
+Kubernetes defines several ways to attach Volumes to Pods. 
+
+The best practice consists in decoupling the need for storage ([Persistent Volume Claims](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#persistentvolumeclaims): PVC) from the real storage ([Persistent Volume](https://kubernetes.io/docs/concepts/storage/persistent-volumes/): PV). The latter is provisioned by Kubernetes administratore based on the available resources. PV can be one of the [several supported](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#types-of-persistent-volumes).
+
+Cloud providers define, in their Kubernetes cluster, mechanisms to allocate PVs based on existing PVCs using their storage infrastructure; on-prem cluster must provision their PV according the existing capability of the system.
+
+In development clusters (e.g. Minikube) or single node clusters, PVCs can be binded to HostPath PVs: a particular kind of volume that maps the Pod's Volume to a directory of the filesystem.
+
+This approach, however, is not a feasible solution in multi-node clusters. A Pod, in fact, can be on different nodes during its lifecycle: Kubernetes can kill and restart it on another node at any time based on the resources available in the cluster. In this scenario, the migrated Pod will not find its old data after restarting on a new node.
+
+A common solution is to use a remote volume mounted in the Pods so that it can be reached from any node in the cluster. 
+
+In this document we will show you:
+
+* how to associate a remote NFS volume to KubeFlow Pods so you don't have any problems when Pod is scheduled on different cluster nodes during its lifecycle. 
+* how simplify the life of the cluster administrator using [dynamic provisioning](https://kubernetes.io/docs/concepts/storage/dynamic-provisioning/) and leaving to Kubernetes the task to create different PV for each PVC without needs on manual intervention.
+
```

Comment:
I think it would be helpful to the user journey if you added something like the following to the end of this section:

`The best way to avoid PVC problems is to complete both of these steps before you install Kubeflow on your Kubernetes cluster. However, if you have already installed Kubeflow on your cluster and want to resolve PVC problems, complete the steps and then read the [In case of existing Kubeflow installation](#in-case-of-existing-kubeflow-installation) section at the end of this document.`